src/Agent.php

```diff
@@ -37,6 +37,7 @@ class Agent extends Mobile_Detect {
      * @var array
      */
     protected static $additionalBrowsers = array(
+        'Vivaldi'           => 'Vivaldi',
```

Comment:
You have to also add in the Vivaldi Version detection under $additionalProperties in the browser section in order to detect the Vivaldi version 

protected static $additionalProperties = array(
...
//Browsers
...
'Vivaldi'           => 'Vivaldi/[VER]',
);